src/coreclr/dlls/mscoree/mscorwks_ntdef.src

```diff
@@ -31,6 +31,7 @@ EXPORTS
 
         ; mono
         mono_unity_initialize_host_apis
+        unity_log
```

Comment:
I would suggest passing this up into managed when initializing (part of the struct) rather than exporting a function. Managed can call it via a function pointer.

Reply:
Why is that better?